task/01-strings-tasks.js

```diff
@@ -22,7 +22,8 @@
  *   '',  'bb'  => 'bb'
  */
 function concatenateStrings(value1, value2) {
-    throw new Error('Not implemented');
+    const concatenatedString = value1.concat(value2)
+    return concatenatedString
```

Comment:
It would be a bit better to split `return` statements with other parts of the code